src/Integration/Helpers/AbsoluteFilePathLocator.cs

```diff
@@ -45,6 +45,7 @@ public string Locate(string relativeFilePath)
                 throw new ArgumentNullException(nameof(relativeFilePath));
             }
 
+            relativeFilePath = relativeFilePath.Replace("/", "\\");
             relativeFilePath = relativeFilePath.TrimStart('\\');
 
             var absoluteFilePaths = new string[1];
```

Comment:
This class is a wrapper around VS API. It will return null if you give it something with forward slashes.
I think we should either handle it and fix to backslashes, or throw if it's not backslashed.
IMO it's better to fix. WDYT?